tests/plugins/test_parser.py

```diff
@@ -1,5 +1,5 @@
 import dataclasses
-from typing import Any, Dict, List, NamedTuple, Tuple, Type
+from typing import Any, NamedTuple, Type
```

Comment:
`type` is actually a built-in (I just learned this today myself)

Reply:
Ah nice - will update